tests/requirements.txt

```diff
@@ -1,11 +1,11 @@
-flake8==3.5.0
```

Comment:
It's important to know that if you are changing the version of the linter, the results of linting will often change accordingly. It's important to decide if that's what you actually want.

Reply:
no longer in mr.